index.js

```diff
@@ -54,6 +54,41 @@ function monkeypatch() {
     estraverse.VisitorKeys.TypeAlias = TypeAliasKeys;
     return results;
   };
+
+  // monkeypatch escope/referencer
+  var referencerLoc;
+  try {
+    var referencerLoc = Module._resolveFilename("./referencer", escopeMod);
+  } catch (err) {
+    throw new ReferenceError("couldn't resolve escope/referencer");
+  }
+  var referencer = require(referencerLoc);
+
+  // if there are decotators, then visit each
+  function visitDecorators(node) {
+    if (!node.decorators) {
+      return;
+    }
+    for (var i in node.decorators) {
```

Comment:
Would be better to loop over this with `for (var i = 0; i < node.decorators.length; i++)` only because I've had issues in the past where people monkeypatch `Array.prototype` and add enumerable properties...